base/logging_new.go

```diff
@@ -0,0 +1,137 @@
+package base
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	// RFC3339 + 3 digit milli
+	timestampFormat = "2006-01-02T15:04:05.000Z07:00"
```

Comment:
We've got this defined elsewhere as base.ISO8601Format.  If you're thinking about removing that, probably best to do that in this PR.

Reply:
Thanks for the heads up, I didn't spot that constant. I've just referenced it directly from here as it's used elsewhere.